lib/grammers-tl-gen/src/structs.rs

```diff
@@ -60,6 +60,33 @@ fn write_struct<W: Write>(
     _metadata: &Metadata,
     config: &Config,
 ) -> io::Result<()> {
+    if let Some(docs) = &def.docs {
+        writeln!(
+            file,
+            "{}/// {}",
+            indent,
+            docs.documentation.description.replace("\n", "<br>")
+        )?;
+
+        writeln!(
+            file,
+            r#"{}/// generated based on <a href="{link}">{name}</a>"#,
```

Comment:
Is this remark necessary? Perhaps it can be worded differently. I'm not sure how this would look:

```suggestion
             r#"{}/// <em>(documentation extracted from <a href="{link}">{name}</a>)</em>"#,
```